### PR TITLE
guns damage multiplier changes

### PR DESCRIPTION
**Pull request by Tea-Primoris**
## About The Pull Request
Fixing fire rates in #5644 

## Why It's Good For The Game
Fix

## Changelog
:cl:
fix: Guns fire rate are working as intended
/:cl:

### CODE REVIEW

**Review by genessee596 — APPROVED** [review]
ready to merge on test server

File: code/modules/projectiles/gun_firemode.dm [hunk 1]
```diff
@@ -49,6 +49,8 @@
 					var/datum/component/item_upgrade/IU = I.GetComponent(/datum/component/item_upgrade)
 					if(IU.weapon_upgrades[GUN_UPGRADE_FIRE_DELAY_MULT])
 						gun.vars["fire_delay"] *= IU.weapon_upgrades[GUN_UPGRADE_FIRE_DELAY_MULT]
+		else if(propname == "damage_mult_add")
+			gun.damage_multiplier += settings[propname]
```
Comment (TorinoFermic) [inline code/modules/projectiles/gun_firemode.dm:53]: These lines are correctly tabbed ?

Comment (Tea-Primoris) [reply code/modules/projectiles/gun_firemode.dm:53]: Yep

**Review by TorinoFermic — APPROVED** [review]

**Comment by genessee596** [comment]
mergin dis 2 test serb

**Pull request merged**